### DISCUSSION TITLE
How to remove an account?

### DISCUSSION THREAD

**Discussion by teo-tsirpanis** [yhype/yhype-feedback#5] [Q&A] [answered]
I haven't found any way to remove my account, and I persume that revoking access to my GitHub account is not enough. How to remove it?

(not that I don't like Ÿ HŸPE, I created an account just to see what it is, and I don't need it for the moment)

**Comment by antonkomarev** [comment]
Good question. You want to delete all the statistics related to the user?

You can revoke GitHub authorization and your user record will be deleted, but statistics will be saved. I may add feature to delete statistics too.

There is no way to completely disappear from the service because somebody may have you as follower for example and your public profile info will be added to service.

**Reply by teo-tsirpanis** [reply]
I see, thanks for your quick response. Perhaps it's not worth removing the account after all; most info are public either way.

**Reply by antonkomarev** [reply]
Yeah, that's why I didn't added removal. Maybe later you will decide to return ;)

**Comment by antonkomarev** [comment, marked as answer]
If you still want to hide or delete your account you can do it by yourself in [Y HYPE settings page](https://yhype.me/settings)

<img width="1512" height="828" alt="image" src="https://github.com/user-attachments/assets/5c07ce7c-4f14-46b7-af94-f55d96a6d5f7" />

Or write me here. I've implemented admin panel for it.
